src/renderer/utils/tootParser.ts

```diff
@@ -1,3 +1,5 @@
+import { faL } from "@fortawesome/free-solid-svg-icons"
```

Comment:
Are you using this `faL`?

Reply:
No, you can ignore that, probably was added due to some errors I had when I initially tried to run this on my environment

Reply:
This is causing the test to fail. Please remove this line if it is not used.